capstone/backend/scraper/scrapers/scraper.py

```diff
@@ -92,6 +92,19 @@ def get_inventory_info(item):
    
     return inventory_dict
 
+def set_items_unavailable_default(transaction):
+  """ Sets all item availabilities to OUT_OF_STOCK.
+  This function marks items that no longer appear 
+  in stores out of stock, as opposed to their old 
+  availability statuses."""
+  transaction.execute_update(
+    "UPDATE Inventories "
+    "SET ItemAvailability = \'OUT_OF_STOCK\' "
+    "WHERE StoreId IN UNNEST(@stores)",
```

Comment:
pass the list of `stores` as a function arg.

Reply:
I was looking into this, and I'm not sure how to do this with minimal changes to my code, so I decided that it's not worth the time: I'll leave a TODO for later, if possible.